pkg/jsonmessage/jsonmessage.go

```diff
@@ -32,17 +32,29 @@ func (e *JSONError) Error() string {
 // value of the progress made towards Total. Total is the end value describing when
 // we made 100% progress for an operation.
 type JSONProgress struct {
-	terminalFd uintptr
-	Current    int64 `json:"current,omitempty"`
-	Total      int64 `json:"total,omitempty"`
-	Start      int64 `json:"start,omitempty"`
+	terminalFd   uintptr
+	Current      int64 `json:"current,omitempty"`
+	Total        int64 `json:"total,omitempty"`
+	Start        int64 `json:"start,omitempty"`
+	DownloadCost time.Duration
```

Comment:
Needs `json:"downloadCost,omitempty"` tag.
Same applies to `ExtractCost` too.

Reply:
I noticed that some structs use Pascal case, while `JSONProgress` uses all lowercase, I'm a little confused, therefore I use the same naming scheme as the `hidecounts` below.